src/App.tsx

```diff
@@ -17,18 +49,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelect={handleSelectFilter}
+                onInput={handleInputFilter}
+                query={query}
+                onCross={handleClearInputBtn}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length === 0 ? (
```

Comment:
the wrong expression was used here
what if the server todos array is empty? then you will always show `Loader`. you should show the loader only during loading todos
to implement it - create a new state `isLoading` and change it accordingly